### PR TITLE
Add dhcp_class define to create custom DHCP class

### LINKED ISSUE #86: Add define for specifying DHCP classes
Add a dhcp::dhcp_class define that allows new client "class" blocks to be specified.

Based on https://github.com/theforeman/puppet-dhcp/commit/b5f37fd and https://github.com/theforeman/puppet-dhcp/commit/96e9d2be9eed0e0ebe24242d35df8144f6e0e01c.

### PR DESCRIPTION
**Pull request by domcleal**
Fixes #86

### CODE REVIEW

File: manifests/dhcp_class.pp [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# == Define: dhcp::dhcp_class
+#
+define dhcp::dhcp_class (
+  $parameters,
+) {
+
+  unless is_array($parameters) {
```
Comment (bastelfreak) [inline manifests/dhcp_class.pp]: we can go puppet4 only now. Do you want to use new datatypes instead?

**Comment by bastelfreak** [comment]
sorry for the late response @domcleal!

File: manifests/dhcp_class.pp (hunk 1 above)
Comment (domcleal) [inline manifests/dhcp_class.pp]: Gladly, PR updated using data types.

**Comment by bastelfreak** [comment]
thank youuuuuuu

**Pull request merged**